spec/features/events_spec.rb

```diff
@@ -72,8 +74,13 @@
 
     click_link "Death Star's destruction celebration"
     page.should have_link('Email')
+    page.should have_link('Participants')
 
     click_link 'Email'
     page.should have_content "Emails for Death Star's destruction celebration"
+
+    visit event_path(events(:party))
+    click_link 'Participants'
+    page.should have_content "Participants of Death Star's destruction celebration"
```

Comment:
@bgeuken https://github.com/openSUSE/travel-support-program/pull/80/files#diff-f20a1b372768b0b5eebe83bc6aa154f8R87 I am checking here that a tsp member is not able to see a user of a shipment request